### PR TITLE
Visitor can navigate the site

### PR DESCRIPTION
**Pull request by EevanR**
[PT](https://www.pivotaltracker.com/n/projects/2424458)

- generate route for root path
- create nav bar
- add signup, login, home and the restaurant title to navbar
- relevant tests pass

Screen Shot 2019-12-10 at 3 24 47 PM

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get 'products/index'
```
Comment (oliverochman) [inline config/routes.rb]: redundant line of code, please remove it

File: config/routes.rb [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+    root controller: :products, action: :index
```
Comment (oliverochman) [inline config/routes.rb:2]: fix indentation

File: coverage/.resultset.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (oliverochman) [inline coverage/.resultset.json]: you have to remove the coverage files from the git cache.
http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

File: features/step_definitions/basic_steps.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Given("I am on the landing page") do
+   visit root_path  
+end
+
```
Comment (oliverochman) [inline features/step_definitions/basic_steps.rb]: please remove empty lines

File: features/step_definitions/assertions.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Then("I should see {string}") do |content|
+    expect(page).to have_text content
```
Comment (oliverochman) [inline features/step_definitions/assertions.rb]: Please use have_content instead.

File: features/user_can_navigate_the_site.feature [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+   Background: 
+      Given I am on the landing page
+
+   Scenario: Can see navbar
+      Then I should see "Signup"
+      And I should see "Login"
+      And I should see "Menu"
+      And I should see "GRETA'S GREENS"
+
```
Comment (oliverochman) [inline features/user_can_navigate_the_site.feature]: Please remove empty lines of code

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (EevanR) [reply config/routes.rb]: done

File: config/routes.rb (hunk 2 above)
Comment (EevanR) [reply config/routes.rb:2]: done

File: coverage/.resultset.json (hunk 3 above)
Comment (EevanR) [reply coverage/.resultset.json]: done

File: features/step_definitions/basic_steps.rb (hunk 4 above)
Comment (EevanR) [reply features/step_definitions/basic_steps.rb]: done

File: features/step_definitions/assertions.rb (hunk 5 above)
Comment (EevanR) [reply features/step_definitions/assertions.rb]: done

File: features/user_can_navigate_the_site.feature (hunk 6 above)
Comment (EevanR) [reply features/user_can_navigate_the_site.feature]: done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**